servicecontrol/integration/index.md

```diff
@@ -1,21 +1,21 @@
 ---
-title: ServiceControl Integration
-reviewed: 2021-04-06
+title: ServiceControl integration
```

Comment:
We don't really have a convention and goodness knows I don't like unnecessary capitalization but I've been leaving the capitalization as-is for titles.

Reply:
I've been changing it. 😉